account_manager/src/validator/create.rs

```diff
@@ -27,6 +27,11 @@ pub const AT_MOST_FLAG: &str = "at-most";
 pub const WALLET_PASSWORD_PROMPT: &str = "Enter your wallet's password:";
 
 pub fn cli_app<'a, 'b>() -> App<'a, 'b> {
+    #[cfg(unix)]
+    let windows = false;
+    #[cfg(windows)]
+    let windows = true;
```

Comment:
It might be cool to use the `cfg!` macro for the bools, like `let windows = cfg!(windows);`

Reply:
done!